corehq/apps/userreports/reports/view.py

```diff
@@ -327,7 +329,7 @@ def headers(self):
     def get_ajax(self, request):
         try:
             data_source = self.data_source
-            if len(data_source.columns) > 50:
+            if not DISABLE_COLUMN_LIMIT_IN_UCR.enabled(self.domain) and len(data_source.columns) > 50:
```

Comment:
super nit picky but you could avoid a DB hit most of the time if you flipped the order of the conditions to check the column limit first